content/en/docs/installation/supported-releases.md

```diff
@@ -224,6 +224,19 @@ Our testing coverage is:
 [`next`]: https://testgrid.k8s.io/jetstack-cert-manager-next
 [`previous`]: https://testgrid.k8s.io/jetstack-cert-manager-previous
 
+The oldest Kubernetes release supported by cert-manager is 1.16, as we want
+to be supporting most commercial Kubernetes offerings.
+
+|   Vendor   | Oldest Kubernetes release | End of Life |
+| :--------: | :-----------------------: | :---------: |
+| [EKS][eks] |           1.16            | 25 Jul 2021 |
+| [GKE][gke] |           1.17            |  Nov 2021   |
+| [AKS][aks] |           1.18            |  Jun 2021   |
+
```

Comment:
suggestion (non-blocking): would it be good to add a timestamp to this table specifically. this might make it clearer for us when we read the table in the future that it could be out of date and due an upgrade.

```suggestion
|   Vendor   | Oldest Kubernetes Release [1] | End of Life |
| :--------: | :---------------------------: | :---------: |
| [EKS][eks] |             1.16              | 25 Jul 2021 |
| [GKE][gke] |             1.17              |  Nov 2021   |
| [AKS][aks] |             1.18              |  Jun 2021   |

[1] as of 2021-05-19
```

Reply:
(NB: I edited the above and updated the date in the suggestion)

Reply:
Excellent point!! I forgot that we will have to update this soon 😅 Done in b0bbbb2d3eb5d48a2db94ab95eb681ff30e2a608